features/add_needs.feature

```diff
@@ -22,31 +22,31 @@ Feature: Add needs
   Background:
     * I am logged into the system
 
-  Scenario Outline: Add any needs
-    Given a resident
-    When I add needs "<need>"
-    And I submit the add needs form
-    Then the residents list of needs contains "<need>"
-    Examples:
-      | need                          |
-      | Phone triage                  |
-      | Groceries and cooked meals    |
-      | Physical and mental wellbeing |
-      | Financial support             |
-      | Staying social                |
-      | Prescription pickups          |
-      | Book drops and entertainment  |
-      | Dog walking                   |
+#  Scenario Outline: Add any needs
```

Comment:
Is this being fixed by @eddieleeper-nb's branch? If not, can we create a card to make sure we get these working again if we're going to merge it commented out? Don't wanna forget about it.

Reply:
Yes - I'll look at fixing this in the test branch.